en/includes/references/conditional-auth/api-reference.md

```diff
@@ -723,6 +724,32 @@ This function returns the local user associated with the federate username given
       </tbody>
     </table>
 
+### Get masked value
+
+`getMaskedValue(value)`
+
+This utility function returns a masked value for the given input value. This function can be used to mask sensitive content in adaptive authentication script logs by developers.
+
+- **Parameters**
+
+    <table>
+      <tbody>
+        <tr>
+          <td><code>value</code></td>
+          <td>Value needs to be masked.</td>
+        </tr>
+      </tbody>
+    </table>
+
+- **Example**
+
+  For debugging purposes, if there is a need to add some logs including sensitive content (ex: PII) then it can be masked using the above method as follows.
```

Comment:
```suggestion
For debugging purposes, if there is a need to add some logs including sensitive content (ex: PII) then it can be masked using the above method as follows.
```